formats/publication/publisher/details.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "documents",
+    "first_public_at",
+    "government",
+    "political"
+  ],
+  "properties": {
+    "body": {
+      "type": "string"
+    },
+    "documents": {
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
Why are these being rendered by the publishing app? These don't need to be embedded in Govspeak, so I think they should be made available as plain data and rendered by the frontend. Or was there a conversation I missed?